### PR TITLE
修改DataSyncer类任务重试机制

### PR DESCRIPTION
**Pull request by gibck**
Please do not create a Pull Request without creating an issue first.

## What is the purpose of the change

修改重试机制,超过15次之后不再重试,重试时间5s 10s 15s 30s 60s 5m 10m 30m 1h 2h 4h 8h 12h 24h

## Brief changelog

XX

## Verifying this change

XXXX

Follow this checklist to help us incorporate your contribution quickly and easily:

* [ ] Make sure there is a Github issue filed for the change (usually before you start working on it). Trivial changes like typos do not require a Github issue. Your pull request should address just this issue, without pulling in other changes - one PR resolves one issue.
* [ ] Format the pull request title like `[ISSUE #123] Fix UnknownException when host config not exist`. Each commit in the pull request should have a meaningful subject line and body.
* [ ] Write a pull request description that is detailed enough to understand what the pull request does, how, and why.
* [ ] Write necessary unit-test to verify your logic correction, more mock a little better when cross module dependency exist. If the new feature or significant change is committed, please remember to add integration-test in [test module](https://github.com/alibaba/nacos/tree/master/test).
* [ ] Run `mvn -B clean package apache-rat:check findbugs:findbugs -Dmaven.test.skip=true` to make sure basic checks pass. Run `mvn clean install -DskipITs` to make sure unit-test pass. Run `mvn clean test-compile failsafe:integration-test`  to make sure integration-test pass.

### CODE REVIEW

**Comment by Maijh97** [comment]
Please do not change the code style, 
refer to https://github.com/alibaba/nacos/blob/develop/style/codeStyle.md

File: naming/src/main/java/com/alibaba/nacos/naming/consistency/ephemeral/distro/DataSyncer.java [hunk 1]
```diff
@@ -49,41 +49,41 @@
 @Component
 @DependsOn("ProtocolManager")
 public class DataSyncer {
-    
+
```
Comment (KomachiSion) [inline naming/src/main/java/com/alibaba/nacos/naming/consistency/ephemeral/distro/DataSyncer.java:52]: Do not change indent. Please use nacos code style to reformat code.

Comment (gibck) [reply naming/src/main/java/com/alibaba/nacos/naming/consistency/ephemeral/distro/DataSyncer.java:52]: 好的

File: naming/src/main/java/com/alibaba/nacos/naming/misc/GlobalConfig.java [hunk 2]
```diff
@@ -27,46 +27,57 @@
  */
 @Component
 public class GlobalConfig {
-    
+
```
Comment (Maijh97) [inline naming/src/main/java/com/alibaba/nacos/naming/misc/GlobalConfig.java:30]: Please do not change indent.

Comment (gibck) [reply naming/src/main/java/com/alibaba/nacos/naming/misc/GlobalConfig.java]: 好的

**Comment by KomachiSion** [comment]
@gibck your implementation has no problem. But we don't know why you change like this.

In my opinion,  the multiple time to retry sync will cause problems in some situation.

For example, the net has some problem so that the sync task failed for a while. After net recovery, your task may be multiple to several hours. It will cause problem like :
1. tasks will be overstocked in thread pool queue for a long time.
2. the newest change will not be sync in time.

So I think the PR is not very reasonable.

**Comment by KomachiSion** [comment]
Hi @gibck, Any response or any discussion? If no more response, we may close PR after few days.

**Pull request closed without merging**